presto-docs/src/main/sphinx/cache/local.rst

```diff
@@ -0,0 +1,69 @@
+=====================
```

Comment:
Can we have presto local cache as an introduction then talk about alluxio local cache?